tensorboard/context.py

```diff
@@ -32,9 +32,14 @@ class RequestContext(object):
 
     Fields:
       auth: An `AuthContext`, which may be empty but is never `None`.
+      remote_ip: An `ipaddress.IPv4Address` or `ipaddress.IPv6Address`, which may be None.
```

Comment:
Should be “An `ipaddress.IPv4Address` or `ipaddress.IPv6Address`, or `None`”.
Saying “a [non-`None` type], which may be `None`” doesn’t make sense.